docs/en/guides/developer/working-with-json/json-intro.md

```diff
@@ -1,18 +0,0 @@
----
-slug: /en/guides/developer/working-with-json/json-intro
-sidebar_label: Introduction 
-sidebar_position: 1
-description: Introduction to Handling JSON
----
-
-# Handling JSON
```

Comment:
Can we really want to just remove this intro?

Reply:
I will move the info into the other pages, I think the first doc that people see should tell them how to load data, not about all of the options.  At then end of loading data I will point them to the options, including the newer experimental choices.